runtime/wasm/runtest/src/lib.rs

```diff
@@ -173,6 +173,7 @@ mod tests {
         LittleEndian::read_i32(val)
     }
 
+    #[allow(dead_code)]
```

Comment:
If we are not using this function, should we just remove it

Reply:
Done.